src/Makefile.in

```diff
@@ -0,0 +1,32 @@
+CC ?= gcc
+LUAJIT_FLAGS = $(shell pkg-config --cflags --libs luajit)
+LUAJIT_VER = $(shell pkg-config --variable=abiver luajit)
+LUASRC = $(shell find . -regex '[^\#]*\.lua' -printf '%P ')
+
+
+BINDIR = $(DESTDIR)/@DYZBINDIR@
+LUADIR = $(DESTDIR)/@DYZLUADIR@
+
+dyz: dyz.c config.h
+	$(CC)  -o $@ $? $(LUAJIT_FLAGS) -Wl,--no-as-needed -Wl,-E
+
+install-bin: dyz
+	@mkdir -p $(BINDIR)
```

Comment:
Please use `install -m755 -t $(DESTDIR)$(BINDIR) dyz`.

Note that there is no `/` between the expansions of `$(DESTDIR)` and `$(BINDIR)`. See [“GNU Coding Standards: DESTDIR”](https://www.gnu.org/prep/standards/html_node/DESTDIR.html) for the rationale.